backend/entities/Models/PageFilter.cs

```diff
@@ -71,6 +72,19 @@ public virtual bool IsValid()
             return this.Page > 0
                 && this.Quantity > 0;
         }
+
+        /// <summary>
+        /// Convert a start index to a page for paging. // TODO: Should switch to using start index instead of paging because it has a little more control.
+        /// </summary>
+        /// <param name="startIndex"></param>
+        /// <param name="quantity"></param>
+        /// <returns></returns>
+        private int CalcPage(int startIndex, int quantity)
```

Comment:
WFS uses `startIndex` for paging.  Will need to consider switching over to using it in the future.